### PR TITLE
Small but important grammar/editorial fixes

### PR DESCRIPTION
**Pull request by JAG-UK**
A bunch of small but important drafting fixes. 
Shouldn't be any controversial content changes in this one

### CODE REVIEW

File: draft-ietf-scitt-architecture.md
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:154]: > Breadth of access if critical so the Transparency 

Was this intended to be:
Breadth of access **is** critical so the

**Review by SteveLasker — COMMENTED** [review]
Thanks, @JAG-UK
Great edits, thank you. Overall LGTM, with one minor nit/clarification of wording.

File: draft-ietf-scitt-architecture.md
Comment (OR13) [inline draft-ietf-scitt-architecture.md:133]:
```suggestion
Producing a Transparent Statement might be considered a form of notarization.
```

Comment (OR13) [reply draft-ietf-scitt-architecture.md:133]: avoiding normative language, not sure if its as much of a deal as it is at W3C

**Review by OR13 — APPROVED** [review]
my nits can be ignored

**Comment by yogeshbdeshpande** [comment]
@JAG-UK I only see Orie Approval, so I would suggest we wait for other approvers, before merging this.

**Comment by henkbirkholz** [comment]
@JAG-UK, I'd like to see this PR presented in the group for feedback, if that is okay.

**Pull request merged**
